test/boulder-config.json

```diff
@@ -2,7 +2,8 @@
   "syslog": {
     "network": "",
     "server": "",
-    "stdoutlevel": 6
+    "stdoutlevel": 6,
+    "sysloglevel": 4
```

Comment:
@ccppuu no problem. I believe we added that to stop spamming the syslog of development machines, @jsha?